tests/sampletester/Samples.cs

```diff
@@ -147,6 +147,7 @@ public class XamarinFormsTester : SampleTester {
 		static Dictionary<string, SampleTest> test_data = new Dictionary<string, SampleTest> {
 				// Build solution instead of csproj.
 				{ "WebServices/TodoWCF/iOS/TodoWCF.iOS.csproj", new SampleTest { BuildSolution = true, Solution = "WebServices/TodoWCF/TodoWCF.sln" } },
+				{ "UserInterface/Xaminals/Xaminals.iOS/Xaminals.iOS.csproj", new SampleTest { BuildSolution = true, Solution = "UserInterface/Xaminals/Xaminals.sln" } },
```

Comment:
Why do you need this change? If you don't bump this repo, you shouldn't need to change anything to make it green (because it's been green in the past).

Reply:
Looks like I misunderstood how this worked at first glance :) you are right, this is not needed